common/views/components/VenueClosedPeriods/VenueClosedPeriods.js

```diff
@@ -0,0 +1,64 @@
+import { useContext } from 'react';
+
+import {
+  backfillExceptionalVenueDays,
+  getExceptionalOpeningPeriods,
+  getExceptionalClosedDays,
+  groupConsecutiveDays,
+  convertJsonDateStringsToMoment,
+} from '../../../services/prismic/opening-times';
+import { formatDayDate } from '@weco/common/utils/format-date';
+import OpeningTimesContext from '@weco/common/views/components/OpeningTimesContext/OpeningTimesContext';
+
+type Props = {|
+  venue: any, // FIXME: Flow
+|};
+
+const VenueClosedPeriods = ({ venue }: Props) => {
+  const openingTimes = useContext(OpeningTimesContext);
+  const exceptionalPeriods = getExceptionalOpeningPeriods(openingTimes);
+  const backfilledExceptionalPeriods = backfillExceptionalVenueDays(
+    convertJsonDateStringsToMoment(venue),
+    exceptionalPeriods
+  );
+  const onlyClosedDays =
+    backfilledExceptionalPeriods &&
+    getExceptionalClosedDays(backfilledExceptionalPeriods);
+  const groupedConsectiveClosedDays = onlyClosedDays
+    ? groupConsecutiveDays(onlyClosedDays)
+    : [];
+
+  return groupedConsectiveClosedDays[0].length > 0 ? (
+    <div className="body-text">
+      <h2>{venue.name} closures</h2>
+      {venue.name.toLowerCase() === 'library' && (
+        <p className="no-margin">
+          Planning a research visit? Our library is closed over bank holiday
```

Comment:
We have a a `locationInformation` prop on places, could we extend it to be something where this is controlled in Prismic?

https://github.com/wellcometrust/wellcomecollection.org/blob/master/prismic-model/js/places.js#L14

Reply:
In another PR?